src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -161,8 +156,8 @@ internal sealed override ImmutableArray<string> NotNullWhenFalseMembers
             // ReturnsVoid property is overridden in this class so
             // returnsVoid argument to MakeFlags is ignored.
             bool isExplicitInterfaceImplementation = property.IsExplicitInterfaceImplementation;
-            this.MakeFlags(MethodKind.PropertyGet, declarationModifiers, returnsVoid: false, isExtensionMethod: false, isNullableAnalysisEnabled: isNullableAnalysisEnabled,
-                isMetadataVirtualIgnoringModifiers: isExplicitInterfaceImplementation && (declarationModifiers & DeclarationModifiers.Static) == 0);
+            this.MakeFlags(MethodKind.PropertyGet, _property.RefKind, declarationModifiers, returnsVoid: false, hasAnyBody: true, isExpressionBodied: true, isExtensionMethod: false, isNullableAnalysisEnabled: isNullableAnalysisEnabled,
```

Comment:
>hasAnyBody: true [](http://example.com/codeflow?start=112&length=16)

This looks like an arbitrary choice

Reply:
If `isExpressionBodied` is true, then this `has any body` since the semantics of `has any body` is 'has a block body or an expression body'.  This follows the semantics everywhere else.

Reply:
> This looks like an arbitrary choice

I see that this constructor is used for an expression bodied accessors.